test/tools/unified-spec-runner/entities.ts

```diff
@@ -290,12 +299,16 @@ export class EntitiesMap<E = Entity> extends Map<string, E> {
   getEntity(type: 'bucket', key: string, assertExists?: boolean): GridFSBucket;
   getEntity(type: 'cursor', key: string, assertExists?: boolean): AbstractCursor;
   getEntity(type: 'stream', key: string, assertExists?: boolean): UnifiedChangeStream;
+  getEntity(type: 'clientEncryption', key: string, assertExists?: boolean): ClientEncryption;
   getEntity(type: EntityTypeId, key: string, assertExists = true): Entity {
     const entity = this.get(key);
     if (!entity) {
       if (assertExists) throw new Error(`Entity '${key}' does not exist`);
       return;
     }
+    if (type === 'clientEncryption') {
+      return entity;
+    }
```

Comment:
```suggestion
    if (type === 'clientEncryption') {
      // we do not have instanceof checking here since csfle might not be installed
      return entity;
    }
```